x-pack/plugins/fleet/server/services/agents/crud.ts

```diff
@@ -220,14 +228,16 @@ async function getAgentDocuments(
 
 export async function getAgentsById(
   esClient: ElasticsearchClient,
-  agentIds: string[],
-  options: { includeMissing?: boolean } = { includeMissing: false }
```

Comment:
is it ok to remove this flag?

Reply:
I believe so. `getAgentDocuments` was only created last week and is only used as a part of this feature. I might have generalized too early, so I deleted the one place it was used and will see if it needs to come back.

Reply:
See https://github.com/elastic/kibana/pull/95024/commits/3f0381cbe6fdf37230b4aa2e91c619eeaf7dd6d5